priv/cabbage/apps/itest/test/itest/transactions.exs

```diff
@@ -0,0 +1,174 @@
+          data = [{key, initial_balance}]
+
+          {:ok, receipt_hash} =
+            amount
+            |> Currency.to_wei()
+            |> Client.deposit(alice_account, Itest.PlasmaFramework.vault(Currency.ether()))
+
+          gas_used = Client.get_gas_used(receipt_hash)
+          key = String.to_atom("alice_gas_used_#{index}")
+          data = [{key, gas_used} | data]
+
+          balance_after_deposit = Itest.Poller.eth_get_balance(alice_account)
+
+          key = String.to_atom("alice_ethereum_balance_#{index}")
+
+          data = [{key, balance_after_deposit} | data]
+          Map.put(state, String.to_atom("alice_data_#{index}"), data)
+        end,
+        timeout: 60_000,
+        on_timeout: :kill_task,
+        max_concurrency: @num_accounts
+      )
+      |> Enum.map(fn {:ok, result} -> result end)
+      |> Enum.reduce(%{}, fn state, acc -> Map.merge(state, acc) end)
+
+    {:ok, state}
+  end
+
+  defthen ~r/^they should have "(?<amount>[^"]+)" ETH on the child chain$/,
+          %{amount: amount},
+          %{alices: alices} = state do
+    alices
+    |> Enum.with_index()
+    |> Task.async_stream(
+      fn {{alice_account, alice_pkey}, index} ->
+        geth_block_every = 1
+
+        {:ok, response} =
+          WatcherSecurityCriticalAPI.Api.Configuration.configuration_get(WatcherSecurityCriticalAPI.Connection.new())
+
+        watcher_security_critical_config =
+          WatcherSecurityCriticalConfiguration.to_struct(Jason.decode!(response.body)["data"])
+
+        finality_margin_blocks = watcher_security_critical_config.deposit_finality_margin
+        to_miliseconds = 1000
+
+        finality_margin_blocks
+        |> Kernel.*(geth_block_every)
+        |> Kernel.*(to_miliseconds)
+        |> Kernel.round()
+        |> Process.sleep()
+
+        expecting_amount = Currency.to_wei(amount)
+
+        balance = Client.get_balance(alice_account, expecting_amount)
+
+        balance = balance["amount"]
+        assert_equal(expecting_amount, balance, "For #{alice_account}")
+      end,
+      timeout: 60_000,
+      on_timeout: :kill_task,
+      max_concurrency: @num_accounts
+    )
+    |> Enum.map(fn {:ok, result} -> result end)
+
+    {:ok, state}
+  end
+
+  defwhen ~r/^they send others "(?<amount>[^"]+)" ETH on the child chain$/,
+          %{amount: amount},
+          %{alices: alices, bobs: bobs} = state do
+    alices
+    |> Enum.zip(bobs)
+    |> Enum.with_index()
+    |> Task.async_stream(
+      fn {{{alice_account, alice_pkey}, {bob_account, _bob_pkey}}, _index} ->
+        {:ok, [sign_hash, typed_data, _txbytes]} =
+          Client.create_transaction(
+            Currency.to_wei(amount),
+            alice_account,
+            bob_account
+          )
+
+        # Alice needs to sign 2 inputs of 1 Eth, 1 for Bob and 1 for the fees
+        submitted = Client.submit_transaction(typed_data, sign_hash, [alice_pkey, alice_pkey])
+        Logger.warn("submitted #{inspect(submitted)}")
```

Comment:
That line probably should be deleted. If we want to log it, then probably debug or info would do.